tools/lightning/errors.md

```diff
@@ -74,3 +74,23 @@ Try the latest version! Maybe there is new speed improvement.
 1. Fix the schema so that the file is entirely in either UTF-8 or GB-18030.
 
 2. Manually `CREATE` the affected tables in the target database, and then set `[mydumper] no-schema = true` to skip automatic table creation.
+
+3. Set `[mydumper] character-set = "binary"` to skip the check. Note that this may introduce mojibake into the target database.
+
+## [sql2kv] sql encode error = [types:1292]invalid time format: '{1970 1 1 0 45 0 0}'
+
+**Cause**: A table contains a column with type `timestamp`, and the time value itself does not exist, either due to DST changes or has exceeded the supported range (1970 Jan 1st to 2038 Jan 19th).
```

Comment:
```suggestion
**Cause**: A table contains a column with the `timestamp` type, but the time value itself does not exist. This is either because of DST changes or (need a subject here) has exceeded the supported range (1970 Jan 1st to 2038 Jan 19th).
```